lambdas/process-async-cri-credential/src/main/java/uk/gov/di/ipv/core/processasynccricredential/helpers/AsyncCriResponseHelper.java

```diff
@@ -33,14 +34,14 @@ public static BaseAsyncCriResponse getAsyncResponseMessage(String criResponseMes
         }
         if (criResponseMessageDto.getError() == null) {
             return SuccessAsyncCriResponse.builder()
-                    .credentialIssuer(criResponseMessageDto.getCredentialIssuer())
+                    .credentialIssuer(Cri.fromId(criResponseMessageDto.getCredentialIssuer()))
```

Comment:
I'm pretty suspicious of this tbh - @Wynndow do you know where this is spec'd (I can ask David/Kiwi if not).

The property in the the DTO is `iss`, which is strongly indicative of an 'issuer' field (i.e. the URI-formatted component id), rather than Cri ID (i.e. 'f2f').

As far as I can tell, F2F isn't actually supplying this field at all, and it defaults to F2F (as per logic above), so it might all be fine, but it would be good to work out what it's supposed to be and handle it properly.

Reply:
I don't I'm afraid - i'm not too clued up on the async return stufff

Reply:
Thanks @Joe-Edwards-GDS - I've removed it and defaulting to F2F